### PR TITLE
PDF Viewer Updates

### PR DESCRIPTION
**Pull request by epetrow**
Releted to: https://github.com/telerik/kendo-themes-private/issues/5

### CODE REVIEW

**Review by TeyaVes — APPROVED** [review]

**Review by Juveniel — APPROVED** [review]

File: tests/pdf-viewer/pdf-viewer-search-panel.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                </button>
+                <button class="k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                    <span class="k-button-icon k-icon k-i-print"></span>
+                </button>
+                <button class="k-button k-button-md k-button-flat k-button-flat-base k-rounded-md k-icon-button">
+                    <span class="k-button-icon k-icon k-i-fullscreen"></span>
+                </button>
+            </div>
+            <div class="k-canvas k-pdf-viewer-canvas k-pos-relative k-overflow-auto">
+                <div class="k-search-panel k-pos-absolute k-top-center">
```
Comment (Stamo-Gochev) [inline tests/pdf-viewer/pdf-viewer-search-panel.html:51]: When the `k-pdf-viewer-canvas` element is scrolled (which is the case if you scroll the document or click on next search match), the search panel is no longer visible:
pdf-viewer-search-scroll

**Review by Stamo-Gochev — COMMENTED** [review]
Suggestions below:

File: tests/pdf-viewer/pdf-viewer-search-panel.html (hunk 1 above)
Comment (Stamo-Gochev) [reply tests/pdf-viewer/pdf-viewer-search-panel.html:51]: On a smaller screen the matches words are somewhat misaligned:
image
seems like we can actually look ok even on this screen size, but the text is not stretched.

Another view
image

I am using http://192.168.100.4:8080/tests/pdf-viewer/pdf-viewer-search-panel.html for testing.

File: tests/pdf-viewer/pdf-viewer-blank-page-blazor.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <div class="k-canvas k-pdf-viewer-canvas">
+                <div class="k-page k-blank-page">
+                    <span class="k-icon k-i-folder-open"></span>
+                    <div class="k-widget k-upload k-upload-async">
+                        <div class="k-dropzone">
+                            <button class="k-upload-button k-button k-button-md k-button-solid k-button-solid-base k-rounded-md">
+                                <span class="k-button-text">Select files...</span>
+                            </button>
+                        </div>
+                    </div>
```
[the remark this answers is not in the document]
Comment (Juveniel) [reply tests/pdf-viewer/pdf-viewer-blank-page-blazor.html:59]: The actual rendering of the FileSelect is the same as the Upload. There are no specific FileSelect selectors. However, we will tweak the main element and remove the extra classes ( k-widget, k-upload-async ).

[the remark this answers is not in the document]
Comment (Stamo-Gochev) [reply tests/pdf-viewer/pdf-viewer-blank-page-blazor.html:59]: I thought that FileSelect had a `k-file-select` class, but appears not to - not sure if this is correct though as they are actually different components. 

As far as the issue is concerned, it is actually OK.

**Review by Stamo-Gochev — APPROVED** [review]

**Pull request merged**